### ISSUE TITLE
Greater than should be less than - validation loss

### ISSUE THREAD

**Issue by tfy14esa** [JiaRenChang/PSMNet#186]
https://github.com/JiaRenChang/PSMNet/blob/a5c27041b34785bcaaf3589cf9724428053e60f7/finetune.py#L174

Hi,
I believe that there is a typo in the finetune.py script. The 3-px error is computed as
`1-(float(torch.sum(correct))/float(len(index[0]))` 

Thus, the error is best when it is lowest (i.e. 0) when 
`float(torch.sum(correct))/float(len(index[0]) = 1`

Thus, we should minimize this and not maximize as it is currently coded. Do you agree? I.e. replace 
`if total_test_loss/len(TestImgLoader)*100 > max_acc:`

with

`if total_test_loss/len(TestImgLoader)*100 < max_acc:`

Perhaps one should rename the `max_acc`-variable to 'lowest_err' or something.

It is also important to put this variable to something high at first e.g. `lowest_err = 1000`. Currently, `max_acc` is set to 0 at the beginning.

**Comment by JiaRenChang** [comment]
Hi, @tfy14esa
Indeed, the name of variable is misleading.
We will modify it. Thanks you!
